java-plugin/src/test/files/AvoidSpringRepositoryCallInLoopWithStream.java

```diff
@@ -0,0 +1,67 @@
+package fr.greencodeinitiative.java.checks;
```

Comment:
please add this kind of test file in java-test-project repository (which is a independent test project to verify all is ok when the full system is deployed locally)